docs/ui/vim.rst

```diff
@@ -1,6 +1,26 @@
-Editing lambdapi source code with `Vim <https://www.vim.org/>`__
-----------------------------------------------------------------
+Editing lambdapi source code with `Vim <https://www.vim.org/>`_
+===============================================================
 
-The ``Vim`` mode can be installed optionally using the command
-``make install_vim`` in the ``lambdapi`` repository. It does not have
-support for the LSP server yet.
+A minimal Vim mode is provided to edit Lambdapi files. It provides
+syntax highlighting and abbreviations to enter unicode characters.
+
+Installation
+------------
+
+Installing from sources
+
+  The Vim mode can be installed using the command
+  ``make install_vim`` in the ``lambdapi`` repository.
+
+Installing with Opam
+
+  If Lambdapi is installed with Opam or using ``dune build`` from the
+  sources, then the line
+
+  .. code:: vim
+
+     set rtp+=~/.opam/OCAML_VERSION/share/vim
```

Comment:
shouldn't we write `$OPAM_SWITCH_PREFIX/share/vim` instead?

Reply:
I didn't know that `$OPAM_SWITCH_PREFIX` was defined, thanks for the tip.